config.toml

```diff
@@ -0,0 +1,333 @@
+withouts = ["tests", "websupport"]
+
+[[bconds.python-jedi]]
+withouts = ["tests"]
+
+[[bconds.python-dateutil]]
+withouts = ["tests"]
+
+[[bconds.python-jsonschema]]
+withouts = ["tests"]
+
+[[bconds.python-sphinxcontrib-websupport]]
+withouts = ["optional_tests"]
+
+[[bconds.python-soupsieve]]
+withouts = ["tests"]
+
+[[bconds.python-towncrier]]
+withouts = ["tests"]
+
+[[bconds.python-pytest-asyncio]]
+withouts = ["tests"]
+
+[[bconds.python-pytest-cov]]
+withouts = ["tests"]
+
+[[bconds.python-flit]]
+withouts = ["tests"]
+
+[[bconds.python-async-timeout]]
+withouts = ["tests"]
+
+[[bconds.python-trio]]
+withouts = ["tests"]
+
+[[bconds.python-Automat]]
+withouts = ["tests"]
+
+[[bconds.python-invoke]]
+withouts = ["tests"]
+
+[[bconds.python-jupyter-client]]
+withouts = ["doc","tests"]
+
+[[bconds.python-matplotlib]]
+withouts = ["check"]
+
+[[bconds.ipython]]
+withouts = ["check","doc"]
+
+[[bconds.python-ipykernel]]
+withouts = ["intersphinx", "tests"]
+
+[[bconds.pybind11]]
+withouts = ["tests"]
+
+[[bconds.python-nbconvert]]
+withouts = ["check", "doc"]
+
+[[bconds.python-nbclient]]
+withouts = ["check"]
+
+[[bconds.python-pyquery]]
+withouts = ["tests"]
+
+[[bconds.python-cherrypy]]
+withouts = ["tests"]
+
+[[bconds.freeipa-healthcheck]]
+withouts = ["tests"]
+
+[[bconds.python-zbase32]]
+withs = ["bootstrap"]
+
+[[bconds.python-Traits]]
+withs = ["bootstrap"]
+
+[[bconds.python-lit]]
+withouts = ["check"]
+
+[[bconds.python-pcodedmp]]
+withs = ["bootstrap"]
+
+[[bconds.python-libcst]]
+withouts = ["tests"]
+
+[[bconds.python-databases]]
+withs = ["bootstrap"]
+
+[[bconds.python-molecule]]
+withouts = ["doc"]
+
+[[bconds.scipy]]
+withouts = ["pythran"]
+
+[[bconds.python-pandas]]
+withs = ["bootstrap"]
+
+[[bconds.grpc]]
+withs = ["bootstrap"]
+
+[[bconds.python-zope-interface]]
+withouts = ["docs"]
+
+[[bconds.python-tqdm]]
+withouts = ["tests"]
+
+[[bconds.python-cryptography]]
+withouts = ["tests"]
+
+[[bconds.python-decopatch]]
+withouts = ["tests"]
+
+[[bconds.python-geopandas]]
+withouts = ["tests"]
+
+[[bconds.python-astropy]]
+withouts = ["check"]
+
+[[bconds.python-pyerfa]]
+withouts = ["tests"]
+
+[[bconds.python-fsspec]]
+withs = ["bootstrap"]
+
+[[bconds.python-pyface]]
+withs = ["bootstrap"]
+
+[[bconds.python-oletools]]
+withs = ["bootstrap"]
+
+[[bconds.python-google-api-core]]
+withouts = ["tests"]
+
+[[bconds.python-googleapis-common-protos]]
+withs = ["bootstrap"]
+
+[[bconds.python-proto-plus]]
+withouts = ["tests"]
+
+[[bconds.python-networkx]]
+withs = ["bootstrap"]
+
+[[bconds.python-lxml]]
+withouts = ["buildrequire_extras"]
+
+[[bconds.python-constantly]]
+withouts = ["tests"]
+
+#TODO: bconds with replacement fails with:
+# File "/home/thrnciar/python-maint/whatdoibuild/bconds.py", line 35, in bcond_cache_identifier
+#   withouts_id = '-'.join(sorted(bcond_config.get('withouts', [])))
+#                                 ^^^^^^^^^^^^^^^^
+# AttributeError: 'str' object has no attribute 'get'
+#
+#[bconds.gdb]
```

Comment:
Does the problem also happen if you use `[[bconds.gdb]]`(double square brackets) consistently with all the other bconds?

Reply:
That fixed the problem, thanks. I totally missed the missing brackets.